databricks_cli/libraries/distutils.py

```diff
@@ -0,0 +1,101 @@
+# Databricks CLI
+# Copyright 2020 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import Command
+
+from databricks_cli.dbfs.api import DbfsApi
+from databricks_cli.dbfs.dbfs_path import DbfsPath
+from databricks_cli.configure.provider import get_config, ProfileConfigProvider
+from databricks_cli.configure.config import _get_api_client
+from databricks_cli.utils import InvalidConfigurationError
+
+import databricks_cli.sdk.service as service
+
+
+class InstallLibraryCommand(Command):
+    user_options = [
+        ('dbfs-path=', None, "Path of a library starting with dbfs://",
+                             "default: dbfs:/FileStore/jars/{package_name}"),
+        ('cluster-id=', None, "cluster id to distribute it", None),
+        ('cluster-tag=', None, "cluster tag to install library", None),
+        ('cluster-name=', None, "cluster name to distribute it", None),
+        ('databricks-cli-profile=', None, "Databricks CLI profile name", None),
```

Comment:
I think we should have a overwrite flag to be consistent with the databricks fs commands.

Reply:
to overwrite a wheel? makes perfect sense.

what do you think about profile flag - maybe shorten it to `--profile` as well? so the command will be something like `python setup.py databricks_install --cluster-id abcd --profile staging --owerwrite`